iplover-app/www/js/data.js

```diff
@@ -21,7 +21,7 @@ iplover.data = (function(){
 
                 file.createWriter(function(writer){
                     writer.onwrite = function(evt){callback(uuid + '.jpg');};
-                    writer.onerror = function(e){alert('error'+e);};
+                    writer.onerror = function(e){alert('An error occurred while saving the image.\nPlease try again.\n'+e);};
```

Comment:
This function's contents should probably be wrapped in a try-catch to make sure to we don't break anything as we add logging.